src/colvarcomp.h

```diff
@@ -1148,6 +1152,8 @@ class colvar::dihedPC
   virtual  ~dihedPC();
   void calc_value();
   void calc_gradients();
+  /// Re-implementation of cvc::get_gradients() to carry over atomic gradients of sub-cvcs
+  void get_gradients(std::vector<int> const &atom_ids, std::vector<cvm::rvector> &atomic_gradients);
```

Comment:
How about `colvar::alpha_angles`?

Reply:
Actually writing this raised the question of addressing this situation in a more generic way: a CVC that combines several sub-CVCs. I didn't move forward yet because I see a few obstacles to handling that in an elegant way, e.g.: (1) the precise mathematical combination will be specific to each combination CVC, and (2) the dependency system is not designed to handle this kind of "horizontal" dependency (CVC to another CVC), although that could probably be added.

Reply:
It seems a bit of work to derive a general template function for only two exceptions. With one already covered, it's mostly a matter of who deals with the other one.

Reply:
That matter is now settled :-)